### PR TITLE
Fixed arguments warning

### PR DESCRIPTION
**Pull request by robertwijas**
CLAide changed the way arguments are defined.
Fix removes following warning:

```
[!] The specification of arguments as a string has been deprecated Pod::Command::Open: `[-a]`
```

### CODE REVIEW

**Comment by ealeksandrov** [comment]
:+1:

File: lib/pod/command/open.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Open < Command
         Opens the workspace in xcode. If no workspace found in the current directory,
         looks up until it finds one. Pass `-a` flag if you want to open in AppCode.
       DESC
-      self.arguments = '[-a]'
+      self.arguments = [['[-a]', :optional]]
```
Comment (yas375) [inline lib/pod/command/open.rb]: @robertwijas thank for the fix! Just one thing I've noted. With current implementation the result of `pod open --help` is looks like:


_Note extra square brackets._

Let's change this line to just `self.arguments = [['-a', :optional]]`.



Thanks!

**Comment by yas375** [comment]
According to CocoaPods/CLAide#33 looks like this needs to be changed a little more...

File: lib/pod/command/open.rb (hunk 1 above)
Comment (robertwijas) [inline lib/pod/command/open.rb]: Fixed.

**Comment by robertwijas** [comment]
@yas375 That's true but this has not been released yet.
If you want to get rid of the warning in the latest CocoaPods version, my change is the proper solution AFAIK.

**Comment by yas375** [comment]
@robertwijas agree.

**Comment by leshill** [comment]
Hi @robertwijas,

Thanks!

**Pull request merged**